sqlmesh/core/snapshot/definition.py

```diff
@@ -1003,6 +1028,17 @@ def disable_restatement(self) -> bool:
         """Is restatement disabled for the node"""
         return self.is_model and self.model.disable_restatement
 
+    @property
+    def model_fqt(self) -> t.Optional[exp.Table]:
+        if not self.is_model:
+            return None
+        node = t.cast(Model, self.node)
+        return node.fqt
+
+    @property
+    def fqn(self) -> str:
```

Comment:
Sometimes it's `fqn`, other times it's `model_fqn` all within the snapshot realm. I argue we should just stick to `fqn`.